Jellyfin.Data/Entities/ActivityLog.cs

```diff
@@ -0,0 +1,150 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Data.Entities
+{
+    public partial class ActivityLog
+    {
+        partial void Init();
+
+        /// <summary>
+        /// Default constructor. Protected due to required properties, but present because EF needs it.
+        /// </summary>
+        protected ActivityLog()
+        {
+            Init();
+        }
+
+        /// <summary>
+        /// Replaces default constructor, since it's protected. Caller assumes responsibility for setting all required values before saving.
+        /// </summary>
+        public static ActivityLog CreateActivityLogUnsafe()
+        {
+            return new ActivityLog();
+        }
+
+        /// <summary>
+        /// Public constructor with required data
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="type"></param>
+        /// <param name="userId"></param>
+        /// <param name="datecreated"></param>
+        /// <param name="logSeverity"></param>
+        public ActivityLog(string name, string type, Guid userId)
+        {
+            if (string.IsNullOrEmpty(name))
+            {
+                throw new ArgumentNullException(nameof(name));
+            }
+
+            if (string.IsNullOrEmpty(type))
+            {
+                throw new ArgumentNullException(nameof(type));
+            }
+
+            this.Name = name;
+            this.Type = type;
+            this.UserId = userId;
+            this.DateCreated = DateTime.UtcNow;
+            this.LogSeverity = LogLevel.Trace;
+
+            Init();
+        }
+
+        /// <summary>
+        /// Static create function (for use in LINQ queries, etc.)
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="type"></param>
+        /// <param name="userId"></param>
+        /// <param name="datecreated"></param>
+        /// <param name="logseverity"></param>
+        public static ActivityLog Create(string name, string type, Guid userId)
+        {
+            return new ActivityLog(name, type, userId);
+        }
+
+        /*************************************************************************
+         * Properties
+         *************************************************************************/
+
+        /// <summary>
+        /// Identity, Indexed, Required
```

Comment:
These can be specified here, but they should come after the standard summary text enforced by our stylecop rules. This applied to all the other properties below as well.
```suggestion
        /// Gets or set the id. Identity, Indexed, Required.
```

Reply:
My original intention was to document the generated entities in a separate PR, but if you feel it is a blocker I can do it here

Reply:
Not all of the entities, just this one. This file is added in this PR so might as well document it here as well.